File: Assets/ShootingRPGKit/Scripts/OtherScript/HealingPickup.js
#pragma strict
var hpRecover : int = 20;
var mpRecover : int = 0;
private var master : Transform;

function OnTriggerEnter (col : Collider) {

	if(col.tag == "Player"){
		col.GetComponent(Status).Heal(hpRecover , mpRecover);
		master = transform.root;
    	Destroy(master.gameObject);
	}
}

File: Assets/ShootingRPGKit/CsharpExample/ScriptCS/OtherScript/HealingPickup.cs
using UnityEngine;
using System.Collections;

public class HealingPickup : MonoBehaviour {
	public int hpRecover = 20;
	public int mpRecover = 0;
	private Transform master;
	
	void  OnTriggerEnter (Collider col){
		if(col.tag == "Player"){
			col.GetComponent<Status>().Heal(hpRecover , mpRecover);
			master = transform.root;
			Destroy(master.gameObject);
		}
	}
}